src/stories/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { ButtonProps } from "./interfaces";
+
+const PrimaryButton = (props: ButtonProps) => {
+  const { onClick, classes, children } = props;
```

Comment:
The prop for the element's class name is usually called `className` for consistency. In case you want to add multiple class names, consider using `classnames` or `clsx`